Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -1056,3 +1056,7 @@
 #if __has_include(<os/lock.h>) && !defined(ENABLE_UNFAIR_LOCK)
 #define ENABLE_UNFAIR_LOCK 1
 #endif
+
+#if !defined(ENABLE_SPATIAL_IMAGE_DETECTION) && PLATFORM(VISION)
+#define ENABLE_SPATIAL_IMAGE_DETECTION 0
+#endif
```

Comment:
You can actually just omit this entirely. `ENABLE()` will automatically evaluate to 0 for anything that isn't defined.